Vagrant/Vagrantfile

```diff
@@ -4,71 +4,54 @@ Vagrant.configure("2") do |config|
     cfg.vm.box = "bento/ubuntu-20.04"
     cfg.vm.hostname = "logger"
     cfg.vm.provision :shell, path: "bootstrap.sh"
-    cfg.vm.network :private_network, ip: "192.168.38.105", gateway: "192.168.38.1", dns: "8.8.8.8"
```

Comment:
You should check that after removing `private_network`, the zeek is not capturing packets from the underlying host OS. For example, if you are running Vagrant on your laptop, it should not capture laptop's OS network packets. 

If it is capturing those packets, then maybe by using both private and public networks, we can solve the issue.

Reply:
ok I will check